src/MoBi.UI/Views/BaseModuleContentView.cs

```diff
@@ -75,6 +75,12 @@ protected override void SetActiveControl()
 
       public void DisableDefaultMergeBehavior() => mergeBehaviorItem.Enabled = false;
 
+      public void HideDefaultMergeBehaviorGroup()
+      {
+         mergeBehaviorGroup.Visibility = LayoutVisibility.Never;
+         this.Height -= mergeBehaviorGroup.Height;
```

Comment:
Can you check if this is necessary? If it is, remove `this` and if not remove the line.